src/CodeCoverageToLcovConverterTests/CodeCoverageReaderTests.cs

```diff
@@ -0,0 +1,77 @@
+using CodeCoverageToLcovConverter;
```

Comment:
nit: usings inside namespace

Reply:
any particular reason for this?

Reply:
Yes there is: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/3a3d5da50ce0678d43f20757be63a3c61b5d8ffd/documentation/SA1200.md